burr/tracking/common/models.py

```diff
@@ -81,6 +82,21 @@ def from_transition(transition: Transition) -> "TransitionModel":
         )
 
 
+class PointerModel(IdentifyingModel):
```

Comment:
can you comment what this model is for? I'm confused which and where it's used compared to the one in common/types.py